src/pages/permissions/forms/GroupPanelForm.tsx

```diff
@@ -0,0 +1,48 @@
+import { FC, ReactNode } from "react";
+import { Form, Input } from "@canonical/react-components";
+import { FormikProps } from "formik/dist/types";
+import AutoExpandingTextArea from "components/AutoExpandingTextArea";
+
+export interface GroupFormValues {
+  name: string;
+  description: string;
+}
+
+interface Props {
+  formik: FormikProps<GroupFormValues>;
+}
+
+const GroupPanelForm: FC<Props> = ({ formik }) => {
```

Comment:
ah right, I named it this way previously since we might have a similar form in the group details page. Since we no longer have detail page for groups, this can simply be `GroupForm`.